app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -544,4 +535,14 @@ class PatientSummaryScreenController @Inject constructor(
           .or(medicalHistoryChangedSince)
     }.blockingGet()
   }
+
+  @WorkerThread
```

Comment:
We are already using strict mode and room to ensure these functions are not called on the main thread, do we still need the annotation?

Reply:
This is a good point, I just started adding them out of habit since this is no longer running on Room's internal thread pool.

I still would like to start adding it out of habit since we might add methods that do not use Room in the repository (say file I/O), and would want this to be a convention that is followed.